### ISSUE TITLE
elm-format converts 'TrueType' to 'True Type' in some situations

### ISSUE THREAD

**Issue by ianmackenzie** [avh4/elm-format#264]
For example, in the following code

```elm
module Test exposing (..)


type ExpressionType
    = TrueType
    | FalseType
    | NotType
    | AndType
    | OrType


expressionTypes =
    [ TrueType, FalseType, NotType, AndType, OrType ]


typename : ExpressionType -> String
typename expressionType =
    case expressionType of
        TrueType ->
            "True"

        FalseType ->
            "False"

        NotType ->
            "Not"

        AndType ->
            "And"

        OrType ->
            "Or"
```

the `TrueType` and `FalseType` get turned into `True Type` and `False Type` within the `expressionTypes` list but not within the `typename` function.

**Comment by ianmackenzie** [comment]
I'm using `elm-format-0.17 0.4.0-alpha` on Windows.

**Comment by avh4** [comment]
Ah, good catch; thanks for reporting!

**Comment by avh4** [comment]
Fixed in 6a39ec0...c819da2.  Will be in the 0.5.0-alpha release.  Thanks!

**Comment by ianmackenzie** [comment]
Wow, quick work, thanks! Looking forward to 0.5 =)
